src/service/resources-impl.js

```diff
@@ -771,10 +777,16 @@ export class Resources {
           resize = true;
         } else if (diff < 0) {
           // 6. The new height is smaller than the current one.
+          if (request.callback) {
+            request.callback(false);
+          }
           resize = false;
         } else {
           // 7. Element is in viewport don't resize and try overflow callback
           // instead.
+          if (request.callback) {
+            request.callback(false);
+          }
           request.resource.overflowCallback(/* overflown */ true,
```

Comment:
So, `overflowCallback` is related to resize requests, but not equal to it. `overflowCallback` specifically means - the element is currently overflowing, e.g. the current boundaries do not fit the whole element. As such, I believe it's definition and existence justified. We just need to make it clearly separate.